project_one/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>project one</title>
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+</head>
+<body>
+    <a id= "grading-element-two" href ="about.html">About</a>
+    <a href = "index.html">Home</a>
+    <div id= "heading">
+        <h1>Project One</h1>
+
+    </div>
+    <br>
+    <br>
+    <button id="grading-element-one" onclick="addingText()">Your DOM will suffer consequences</button>
+
+    <div id= "grading-element-one-consequence">

```

Comment:
should not be in the `index.html`...create & attach to the dom when `addingText()` fires, please.